config/locales/en.yml

```diff
@@ -1925,6 +1925,8 @@ en:
         gravatar: "Use Gravatar"
         link: "http://wiki.openstreetmap.org/wiki/Gravatar"
         link text: "what is this?"
+        disabled: "Gravatar disabled!"
+        enabled: "Gravatar enabled!"
```

Comment:
I wonder if these (especially the disabled) could do with more explanation? Probably lose the explanation mark as well...